charts/tfy-karpenter-config/values.yaml

```diff
@@ -93,6 +94,61 @@ karpenter:
     ##
     providerRefName: default
 
+  ## Provisioner for Control plane workloads
+  controlPlaneProvisioner:
+    ## @param karpenter.controlPlaneProvisioner.enabled Enable control plane provisioner for control plane workloads
+    ##
+    enabled: true
+
+    ## @param karpenter.controlPlaneProvisioner.consolidation.enabled Enable consolidation for control plane provisioner
+    ##
+    consolidation:
+      enabled: true
+
+    ## @param karpenter.controlPlaneProvisioner.ttlSecondsAfterEmpty Time (in seconds) after which node will be drained. Either one of consolidation or ttlSecondsAfterEmpty can be used
+    ##
+    ttlSecondsAfterEmpty: 30
+  
+    ## @param karpenter.controlPlaneProvisioner.capacityTypes Capacity types of control plane provisioner
+    ## 
+    capacityTypes:
+      - spot
+      - on-demand
+
+    ## @param karpenter.controlPlaneProvisioner.zones Zones to launch instances for control plane workloads
+    ##
+    zones: []
+
+    instanceFamilies:
+      ## @param karpenter.controlPlaneProvisioner.instanceFamilies.allowed Allowed instance families for control plane workloads
+      ##    
+      allowed: []
+
+      ## @param karpenter.controlPlaneProvisioner.instanceFamilies.notAllowed Not allowed instance families for control plane workloads
+      ##      
+      notAllowed: ["t3", "t2", "t3a", "p2", "p3", "p4d", "p4de", "g4dn", "g5", "g4ad", "inf1", "inf2", "trn1", "trn1n"]
```

Comment:
Is there a reason for not allowing the burstable ones?

Reply:
performance issues with burstable ones